components/setting_picture.jsx

```diff
@@ -335,7 +338,8 @@ export default class SettingPicture extends Component {
                                 />
                             </button>
                             <button
-                                tabIndex={disableSaveButtonFocus ? '-1' : '0'}
```

Comment:
Please revert this change which is for Accessibility.

Reply:
@saturninoabril reverted the changes and updated the pull request

Reply:
@saturninoabril reverted the changes